tv_apps/smart-home/js/card_picker.js

```diff
@@ -83,6 +149,25 @@
     },
 
     show: function(folderElem) {
+      var nameResolvingPromise = null;
+      if (folderElem) {
+        var card = this._cardManager.findCardFromCardList({
+          cardId: folderElem.dataset.cardId
+        });
+        var name = this._cardManager
+          .resolveCardName(card, document.documentElement.lang);
+        nameResolvingPromise = name.raw ?
+          name.raw : document.l10n.formatValue(name.id);
+      } else {
+        nameResolvingPromise = document.l10n.formatValue('my-folder');
+      }
+      Promise.all([nameResolvingPromise]).then(([name]) => {
+        this.input.value = name;
+        this.input.dataset.folderName = name;
+        this.input.setSelectionRange(
```

Comment:
If not set the caret in advance, the caret will not appear for the 1st time of focusing.
Just added this comment.